controllers/auth.controller.js

```diff
@@ -0,0 +1,57 @@
+const passport = require('passport')
+const User = require('../models/user.model')
+
+exports.index = (req, res) => {
+    res.render('../views/welcome', {user: req.user})
+}
+// This function is used to fo loging purpose(Authentication)
+exports.login = (req, res) => {
+    if (req.user)
+        res.status(403).send({"message": "Already logged in. Logout to continue."})
+    else
+        passport.authenticate('local')(req, res, function(){
+            res.redirect('/')
```

Comment:
@shubham1172 Seems doubtful to me. Should redirect to some other endpoint.

Reply:
It's fine, '/' should be the home page only. Middleware will handle the login result.